### ISSUE TITLE
`Fmt_tty.setup_std_outputs` doesn't set the margins

### ISSUE THREAD

**Issue by Drup** [dbuenzli/fmt#11]
Not sure if that's intended.

**Comment by dbuenzli** [comment]
Unclear.

**Comment by Drup** [comment]
http://caml.inria.fr/pub/docs/manual-ocaml/libref/Format.html#VALset_margin

**Comment by dbuenzli** [comment]
I meant *Not sure if that's intended* : unclear. For these reasons:

1. If we assume pure text dump (âlâ cmdliner), then I would certainly like to set the margin if < 80 columns but not if greater because of basic typographic rules for readability.
2. If we assume widgetgiligili, then a) width should be arbitrary b) it should be able to react to changes, which brings us to register formatters for change and I'm not sure this is the right place to do it.

**Comment by dbuenzli** [comment]
> If we assume pure text dump

in a tty.

**Comment by Drup** [comment]
What about providing the appropriate functions but not calling them automatically, then ?

**Comment by dbuenzli** [comment]
> What about providing the appropriate functions but not calling them automatically, then ?

Yeah, but then I feel I have the moral obligation (le problème avec la morale c'est que c'est toujours la morale des autres) to also provide the reactive API.
